.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/AssignmentInCondition:
+  Description: Don't use assignment in conditions.
+  StyleGuide: "#safe-assignment-in-condition"
+  Enabled: true
+  AllowSafeAssignment: true
+
```

Comment:
`(=)` is standing out more than just `=` and `==` so it's a safer choice, and a default one. Since it does not cause problems (other then personal dislikes), let's stick with the default. We can avoid putting more assignments into the conditions or gradually move them out later.